libraries/resource_providers/api200/logical_interconnect_provider.rb

```diff
@@ -82,6 +82,33 @@ def firmware_handler(action)
         end
       end
 
+      def load_ports(item_to_update)
+        port_monitor_attributes = convert_keys(@new_resource.port_monitor, :to_s)
+        enable_port_monitor = item_to_update['portMonitor']['enablePortMonitor'] if item_to_update['portMonitor'] && item_to_update['portMonitor']['enablePortMonitor']
+        enable_port_monitor ||= port_monitor_attributes['enablePortMonitor']
```

Comment:
Also, the way this is setup make it so `data['portMonitor']['enablePortMonitor']` has precedence over `@new_resource.port_monitor['enablePortMonitor']`. 
Is this the expected behavior? If it is, we want to be sure to inform that `data > new_resource` if both are declared